src/Features/CSharpTest/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -8984,4 +8985,303 @@ public class C
             }
             """);
     }
+
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/67270")]
+    public async Task TestTopLevel1()
+    {
+        await TestAsync(
+            """
```

Comment:
>""" [](http://example.com/codeflow?start=12&length=3)

this is a super-nit, but with these long multiline strings in tests, I would generally find it easier to distinguish the strings if named parameters were used.

Reply:
understood.  that will come with more porting to the new sdk test harness.